cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -1011,6 +1011,11 @@ message RefreshLockRequest {
   // REQUIRED.
   // The lock metadata.
   Lock lock = 3;
+  // OPTIONAL.
+  // The existing lock metadata.
+  // This can be used to refresh the lock with a new lock while
+  // ensuring that the lock holder also holds the existing lock
+  Lock existing_lock = 4;
```

Comment:
Makes sense. What about passing `string existing_lock_id` as opposed to the whole `Lock` structure? The `user` or `appname` fields must match in all cases.

Reply:
fine.